pkg/pillar/agentlog/agentlog.go

```diff
@@ -135,7 +141,12 @@ func handleSignals(sigs chan os.Signal) {
 // Print out our stack
 func printStack() {
 	stacks := getStacks(false)
-	log.Errorf("fatal stack trace due to %s:\n%v\n", savedRebootReason, stacks)
+	stackArray := strings.Split(stacks, "\n\n")
```

Comment:
I suspect this gets you only one element since it is only asking for the current goroutine stack (that is what false arg means). But doesn't hurt and makes the code more uniform to do it here as well.